tests/components/camera/test_webrtc.py

```diff
@@ -800,45 +800,6 @@ async def mock_hls_stream_source_fixture() -> AsyncGenerator[AsyncMock]:
         yield mock_hls_stream_source
 
 
-@pytest.mark.usefixtures(
```

Comment:
As camera capabilities are calculated based on implemented methods and not a attribute set by an integration, it's not anymore possible to only set the frontend_stream_type and don't implement the WebRTC methods